sockeye/layers.py

```diff
@@ -659,7 +659,7 @@ def hybrid_forward(self, F, data, steps, weight):  # pylint: disable=arguments-d
         Applies positional embeddings to input data.
 
         :param data: Input data. Shape: (batch, length or 1, num_embed)
-        :param steps: Optional steps input. If given, shape is (batch,)
+        :param steps: Optional steps input. If given, shape is (batch_size or 1, seq_len,)
```

Comment:
parentheses missing?

Reply:
is it? So the first dimension is either be 1 or batch_size. Anyway, this is maybe confusing.